Typos correction/README.md

```diff
@@ -0,0 +1,92 @@
+Baseline model ready. Short description:
```

Comment:
Use markdown formatting in readmes (e.g. sections with `##`, `###`, etc).

Reply:
I find these resources helpful, when need to refresh something about MD
 - https://guides.github.com/features/mastering-markdown/
 - https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

Reply:
Thanks, very helpful